opencti-platform/opencti-graphql/src/domain/work.js

```diff
@@ -197,11 +197,6 @@ export const createWork = async (context, user, connector, friendlyName, sourceI
   return loadWorkById(context, user, workId);
 };
 
-const isWorkCompleted = async (workId) => {
-  const { import_processed_number: pn, import_expected_number: en } = await redisGetWork(workId);
-  return { isComplete: parseInt(pn, 10) === parseInt(en, 10), total: pn };
```

Comment:
you would return also 'expected' now that was not returned before, no issue ?

Reply:
Expected is used by the caller of redisUpdateWorkFigures and not directly by the caller of isWorkCompleted.
I dont think its a problem.